modules/Bio/EnsEMBL/Compara/RunnableDB/NewPerSpeciesComparaDB.pm

```diff
@@ -0,0 +1,99 @@
+=head1 LICENSE
+
+See the NOTICE file distributed with this work for additional information
+regarding copyright ownership.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::RunnableDB::NewPerSpeciesComparaDB
+
+=head1 DESCRIPTION
+
+Creates new MySQL compara database for a single genome_name
+E.g. standaloneJob.pl Bio::EnsEMBL::Compara::RunnableDB::NewPerSpeciesComparaDB \
+    -homology_host mysql-ens-compara-prod-2 \
+    -genome_name canis_lupus_familiaris \
+    -curr_release 103
+
+=cut
+
+package Bio::EnsEMBL::Compara::RunnableDB::NewPerSpeciesComparaDB;
+
+use warnings;
+use strict;
+
+use Bio::EnsEMBL::Compara::DBSQL::DBAdaptor;
+use Bio::EnsEMBL::Compara::Utils::Database qw/ table_exists /;
```

Comment:
It is the first time I see `/` instead of `()` being use here :nerd_face:

Reply:
The possibilities with `qw` are not so delimiting :nerd_face: